### ISSUE TITLE
Issue with IC2 Solar Panels not being removed with wrenches

### ISSUE THREAD

**Issue by LabHouse** [xJon/The-1.12.2-Pack#331]
1. Are you using a legitimate launcher and Minecraft account:
Yes
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
Yes
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
Yes

**Information required**
1. Is the modpack crashing: No
2. Crash/latest log *link* ([Where to find](https://github.com/xJon/The-1.12.2-Pack/blob/master/log_location.md#finding-modpack-installation-folder)): **link to paste**
3. Is Optifine installed or any other additional mods: No

**Describe the issue**

Basically, I cannot wrench my IC2 solar panels in order to remove them. I understand I'm supposed to press SHIFT + RIGHT CLICK (or sneaking and then clicking "use item") to remove it with a wrench but it won't remove it for some reason. No sound is being made and there is no animation

I tried creating a new world in creative mode to troubleshoot. Apparently whether I use a normal wrench or electric wrench, it won't let me wrench a LV, MV, or HV Solar Array...But the wrench will work on a normal tier 1 solar panel and basically any other IC2 device.

Please help. Thanks!

**Comment by xJon** [comment]
Hi, I checked and indeed what you're reporting is true, but I'm not sure if it's a bug or a feature.
I reported it to the author, but for now you can break those solar arrays with a pickaxe.
Thanks anyway for reporting!
